### ISSUE TITLE
Vuepress documentation broken

### ISSUE THREAD

**Issue by PiiXiieeS** [Sanofi-IADC/konviw#295]
**Describe the bug**
Vuepress docs build is broken

**To Reproduce**
`npm run docs:build`

**Expected behavior**
Build as usual

**Screenshots**

```json
Error: Rule can only have one resource source (provided resource and test + include + exclude) in {
  "exclude": [
    null
  ],
  "use": [
    {
      "loader": "cache-loader",
      "options": {
        "cacheDirectory": "/Users/jhgascon/Code/konviw/node_modules/@vuepress/core/node_modules/.cache/vuepress",
     ...
```

**Desktop (please complete the following information):**
- System:
  - OS: macOS 11.1
  - CPU: (16) x64 Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
  - Memory: 318.13 MB / 32.00 GB
  - Shell: 5.8 - /bin/zsh
- Binaries:
  - Node: 16.0.0 - ~/.nvm/versions/node/v16.0.0/bin/node
  - npm: 7.23.0 - ~/.nvm/versions/node/v16.0.0/bin/npm
- Browsers:
  - Chrome: 97.0.4692.71
  - Safari: 14.0.2

**Additional context**
I had recurrent issues with Vuepress in the past and too many extra dependencies that somehow pollute the konviw nestjs repository. 
An alternative would be the to move the docs to another repository or to move vuepress or vitepress into a folder with another package.json removing dependencies risks with konviw.

**Comment by PiiXiieeS** [comment]
@MeStrak do you have any recommendation on the better structure for the documentation and avoiding the complexity of the devDependencies based on the pure documentation package?

**Comment by MeStrak** [comment]
> @MeStrak do you have any recommendation on the better structure for the documentation and avoiding the complexity of the devDependencies based on the pure documentation package?

This is indeed annoying. I would avoid splitting docs to a separate repo - better to have a separate `package.json` if the errors keep happening. Vuepress 2 is also in beta 33 so possibly the right time to begin the migration.

**Comment by MeStrak** [comment]
@PiiXiieeS I noticed in the currently deployed docs that the embedded iFrames don't seem to be working (400 errors). I'm working on separating the package.json - I'll take a look at this too but if you have any quick pointers that would be helpful.

https://sanofi-iadc.github.io/konviw/demoNoTitle.html

![image](https://user-images.githubusercontent.com/31989238/149673227-92bbc3dc-194f-419f-bfed-92350f0b156a.png)

**Comment by MeStrak** [comment]
> @PiiXiieeS I noticed in the currently deployed docs that the embedded iFrames don't seem to be working (400 errors). I'm working on separating the package.json - I'll take a look at this too but if you have any quick pointers that would be helpful.

I found that the issue is with the `nozoom` parameter. It works perfectly on a local version, but does not work on the deployed version on Vercel.

Works:
https://konviw.vercel.app/cpv/wiki/spaces/konviw/pages/32981?type=title&theme=light

Does not work:
https://konviw.vercel.app/cpv/wiki/spaces/konviw/pages/32981?type=title&theme=light&nozoom

Is the deployed version on Vercel up to date with the latest release?

The docs build issue is fixed in this PR: https://github.com/Sanofi-IADC/konviw/pull/296, and I also added a CI script to auto publish new versions of docs and also give visibility of failing docs builds within the PR.

**Comment by PiiXiieeS** [comment]
> > @PiiXiieeS I noticed in the currently deployed docs that the embedded iFrames don't seem to be working (400 errors). I'm working on separating the package.json - I'll take a look at this too but if you have any quick pointers that would be helpful.
> 
> I found that the issue is with the `nozoom` parameter. It works perfectly on a local version, but does not work on the deployed version on Vercel.
> 
> Works: https://konviw.vercel.app/cpv/wiki/spaces/konviw/pages/32981?type=title&theme=light
> 
> Does not work: https://konviw.vercel.app/cpv/wiki/spaces/konviw/pages/32981?type=title&theme=light&nozoom
> 
> Is the deployed version on Vercel up to date with the latest release?
> 
> The docs build issue is fixed in this PR: #296, and I also added a CI script to auto publish new versions of docs and also give visibility of failing docs builds within the PR.

I fixed that.
